test/ered_tls_SUITE.erl

```diff
@@ -0,0 +1,287 @@
+-module(ered_tls_SUITE).
+
+-compile([export_all, nowarn_export_all]).
+
+all() ->
+    [
+     t_command,
+     {group, require_faketime}
+    ].
+
+groups() ->
+    [
+     %% Tests that require 'faketime' to manipulate the system time.
+     {require_faketime, [sequence],
+      [
+       t_expired_cert_tls_1_2,
+       t_expired_cert_tls_1_3
+      ]
+     }].
+
+-define(MSG(Pattern, Timeout),
```

Comment:
These are duplicated. We could put them in a header file that's included in both suites.

Reply:
Done